Add KB article link for error fix (#2195)
theforeman/foreman-documentation: guides/doc-Upgrading_and_Updating/topics/preparing_to_migrate_pulp_content.adoc

To prevent this, pre-migrate Pulp content while running the latest version of {ProjectServer} {ProjectVersionPrevious}.
This reduces the overall upgrade downtime.

During migration to Pulp 3, the amount of data stored in the `/var/lib/pulp/published/` directory can double in size.
Ensure that there is enough space on the `/var/lib/pulp` volume.
If the user runs `{foreman-maintain} prep-6.10-upgrade`, the content in `/var/lib/pulp/content` does not need to be duplicated.

> After Pulp 2 is removed, the extra space can be claimed back, however, shrinking of XFS volumes is not possible and copying data to a different partition might be necessary in order to bring volume size down to the previous value.

Reviewer: ```suggestion
After Pulp 2 is removed, the extra space can be claimed back, however, shrinking of XFS volumes is not possible, and copying data to a different partition might be necessary in order to bring volume size down to the previous value.
```
Rest LGTM!

> After Pulp 2 is removed, the extra space can be claimed back, however, shrinking of XFS volumes is not possible, and copying data to a different partition might be necessary in order to bring volume size down to the previous value.

Reviewer 2: ```suggestion
After Pulp 2 is removed, the extra space can be claimed back.
However, shrinking of XFS volumes is not possible.
Copying data to a different partition might be necessary in order to bring volume size down to the previous value.
```

* Ensure that all enabled repositories complete the synchronization process.
* If any repositories are synced with a *Warning* or *Error* state, fix the underlying problem and resync the repository.

> * If the Pulp 2 to Pulp 3 migration fails with the error `NoMethodError: undefined method `link?' for nil:NilClass`, see the Red Hat Knowledgebase solution https://access.redhat.com/solutions/6898881[The Pulp 2 to Pulp 3 migration fails with error "NoMethodError: undefined method `link?' for nil:NilClass" in Red Hat {ProjectVersionPrevious}] on the customer portal.

Reviewer 2: ```suggestion
* If the Pulp 2 to Pulp 3 migration fails with the error `NoMethodError: undefined method `link?' for nil:NilClass`, see the Red Hat Knowledgebase solution https://access.redhat.com/solutions/6898881[The Pulp 2 to Pulp 3 migration fails with error "NoMethodError: undefined method `link?' for nil:NilClass" in Red Hat Satellite 6.9] on the customer portal.
```
